sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/src/Generated/Models/VMwareCbtMigrationDetails.cs

```diff
@@ -141,6 +152,10 @@ internal VMwareCbtMigrationDetails(string instanceType, string vmwareMachineId,
         public string TargetAvailabilityZone { get; }
         /// <summary> The target proximity placement group Id. </summary>
         public string TargetProximityPlacementGroupId { get; }
+        /// <summary> The confidential VM key vault Id for ADE installation. </summary>
+        public string ConfidentialVmKeyVaultId { get; }
```

Comment:
same. what's the format of this id?

Reply:
`ConfidentialVmKeyVaultId` is now `ResourceIdentifier`.